MDANSE/Src/MDANSE/Framework/Configurators/IConfigurator.py

```diff
@@ -78,6 +79,13 @@ def __str__(self) -> str:
 
 
 class PredictionSettings(NamedTuple):
+    """Strings used for showing the predicted output range in the GUI.
+
+    key - The dictionary key of where the Sequence[float] is stored in the Configurator.
+    label - An explanation for the user of what physical property the numbers represent.
+    unit - The internal unit of MDANSE of the number array. Used for converting to GUI units.
+    """
+
     key: str
     label: str
     unit: str = "none"
```

Comment:
It is strictly possible to document the parameters directly, we'd need to double check the sphinx format we use supports it though.
```suggestion
    """
    
    #: The dictionary key of where the Sequence[float] 
    #: is stored in the Configurator.
    key: str
    #: An explanation for the user of what physical 
    #: property the numbers represent.
    label: str
    #: The internal unit of MDANSE of the number array.
    #: Used for converting to GUI units.
    unit: str = "none"
```